### ISSUE TITLE
Release can't work

### ISSUE THREAD

**Issue by eromoe** [isacikgoz/tldr#6]
Hello,

I downloaded `tldr_0.5.0_linux_arm64.tar.gz`, and found it can not work : 

```
➜  ~ ./tldr --help
usage: tldr [<flags>] [<command>...]

Flags:
      --help         Show context-sensitive help (also try --help-long and --help-man).
  -c, --clear-cache  Clear local repository then clone github.com/tldr-pages/tldr
  -u, --update       Pulls the latest commits from github.com/tldr-pages/tldr
  -s, --static       Static mode, application behaves like a conventional tldr client.
  -r, --random       Random page for testing purposes.
      --version      Show application version.

Args:
  [<command>]  Name of the command. (e.g. tldr grep)

➜  ~ ./tldr -u
➜  ~ ./tldr tar
This page (tar) doesn't exist yet!
Submit new pages here: https://github.com/tldr-pages/tldr#                                                              
➜  ~ ./tldr zip
This page (zip) doesn't exist yet!
Submit new pages here: https://github.com/tldr-pages/tldr#                                                              
➜  ~ ./tldr -r
panic: invalid argument to Intn

goroutine 1 [running]:
math/rand.(*Rand).Intn(0xc4200821b0, 0x0, 0x157c599f75927412)
        /usr/lib/go-1.10/src/math/rand/rand.go:166 +0x9c
math/rand.Intn(0x0, 0x74fc9f)
        /usr/lib/go-1.10/src/math/rand/rand.go:326 +0x37
github.com/isacikgoz/tldr/pkg/pages.QueryRandom(0x0, 0x0, 0x0)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/tldr/pkg/pages/pages.go:90 +0x60e
main.main()
        /home/isacikgoz/git/tldr/main.go:31 +0x567

```

**Comment by eromoe** [comment]
os: ubuntu 18.04 
uname -a : Linux ubuntu 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

I have installed tldr bash version before , did this matter ?

**Comment by eromoe** [comment]
Sorry, I pasted the wrong name .
 It is `tldr_0.5.0_linux_amd64`  , `tldr_0.5.0_linux_arm64 ` even can't execute .

**Comment by isacikgoz** [comment]
Well, let's focus on the repository existence issue.

tldr++ uses `$HOME/.local/share` folder to store `tldr-pages`repository.
Let's check the steps below
- Does `$HOME/.local/share` folder exist? And do you have required permissions on it? (e.g. rw)
- Is there any folder named `$HOME/.local/share/tldr` and is it a git repository?
- Lastly, you can manually clone by running command below:
  `cd $HOME/.local/share/ && git clone https://github.com/tldr-pages/tldr.git`

**Comment by eromoe** [comment]
`$HOME/.local/share/tldr` is a directory but not git repository.
Remove it and `tldr -u`  solve it.

I think it is because previous installed tldr(bash version) generate that folder .

**Comment by isacikgoz** [comment]
I will add a check for this issue in the next release, thanks for the comprehensive report and assisting with the resolution @eromoe!

**Comment by halturin** [comment]
Doesnt work for me too. Trying to build:

```
go get -u github.com/isacikgoz/tldr
```
got error

```
<<skipped>>
go: extracting github.com/pkg/term v0.0.0-20190109203006-aa71e9d9e942
# github.com/isacikgoz/tldr/pkg/prompt
/home/user/devel/goenv/pkg/mod/github.com/isacikgoz/tldr@v0.5.0/pkg/prompt/prompt.go:179:12: confirm.AddSudo undefined (type *survey.Confirm has no field or method AddSudo)
```

**Comment by isacikgoz** [comment]
Thanks for the feedback @halturin I will be looking into this issue. It is probably caused by a dependency issue, for now, you can install pre-compiled binary from releases page.

**Comment by isacikgoz** [comment]
There is no sign of any occurrence of this issue. Hence, I am closing it.
